sdks/swift/Mentat/Mentat/Transact/TxReport.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
Nit: there are these surplus comment lines at the top of some of these files. Please make sure you clean them up when switching licenses.